### ISSUE TITLE
Some questions about Image Normalization

### ISSUE THREAD

**Issue by MakeCent** [open-mmlab/mmaction2#255]
I found "img_norm_cfg" in "tsn_r50_1x1x3_80e_ucf101_rgb.py" and "tsn_r50_1x1x3_100e_kinetics400_rgb.py" are different with each other. They are "mean=[104, 117, 128], std=[1, 1, 1]" and "mean=[123.675, 116.28, 103.53], std=[58.395, 57.12, 57.375]" respectively.
While both of them use resnet50 as the backbone and use pre-trained weights from torchvision. Why use different norm_cfg and how should I determine it?

I also notice that the second img_norm_cfg is equal to the normalization configurations mentioned in torchvison website divided by 255. But I cannot find any pipeline did this operation, i.e. (/255). I cannot found this in Normalize, neither in ToTensor. And when I print the range of image values after ToTensor pipeline, it seems there are lots of images are not in range of (0, 1).

**Comment by innerlee** [comment]
Thanks for the report. You are right, this line https://github.com/open-mmlab/mmaction2/blob/master/configs/recognition/tsn/tsn_r50_1x1x3_80e_ucf101_rgb.py#L28 looks strange. Please change it to the other value

**Comment by innerlee** [comment]
Confirmed with @dreamerlin , this config is modified from mmaction's tsn+bninception https://github.com/open-mmlab/mmaction/blob/master/configs/TSN/ucf101/tsn_rgb_bninception.py#L31 , which originates from the original tsn https://github.com/yjxiong/temporal-segment-networks/blob/2c5e5e7c9c4f552cfde46fa90a4c0c230d857afe/pyActionRecog/action_caffe.py#L27

So the original one does not scale, and the mean values are a little bit different from pytorch's.

**Comment by MakeCent** [comment]
> Confirmed with @dreamerlin , this config is modified from mmaction's tsn+bninception https://github.com/open-mmlab/mmaction/blob/master/configs/TSN/ucf101/tsn_rgb_bninception.py#L31 , which originates from the original tsn https://github.com/yjxiong/temporal-segment-networks/blob/2c5e5e7c9c4f552cfde46fa90a4c0c230d857afe/pyActionRecog/action_caffe.py#L27
> 
> So the original one does not scale, and the mean values are a little bit different from pytorch's.
> 
> We plan to update the normalize cfg to new ones, and release a new checkpoint

Thanks.

**Comment by dreamerlin** [comment]
We may add a new config for TSN-UCF101
